app/src/config/config.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
dont do it here, make a change in the eslintrc.js that is as scoped as possible. avoid these full file disables when possible

Reply:
that seems excessive. is there not a way to allow starting with a number? that seems like all you need

Reply:
looks good. if it passes the action, then good to go